content/CORE/Storage/Pools/StorageEncryption.md

```diff
@@ -50,136 +49,273 @@ These algorithms provide authenticated encryption with block ciphers.
+Type the passphrase and click **SUBMIT**. 
+To unlock child datasets, set the **Unlock Children** checkbox. 
+Child datasets that inherited the parent dataset's encryption settings unlock when the parent unlocks. 
+Users can unlock child datasets with different passphrases as the parent simultaneously by entering their passphrases.
 
-Confirm unlocking the datasets and wait for a dialog to show the unlock is successful.
+Two confirmation windows display. 
+Click **CONTINUE** to confirm you want to unlock the datasets or **CANCEL** to exit and keep the datasets locked. 
+A second confirmation window displays confirming the datasets are unlocked. 
+Click **CONTINUE** to close the second confirmation window. 
 
 ![StoragePoolsDatasetUnlockSuccess](/images/CORE/12.0/StoragePoolsDatasetUnlockSuccess.png "Dataset Unlock Success")
 
-**Example:**
+TrueNAS displays the dataset with the unlocked icon.
+
+**Unlocking Mutliple Datasets Example:**
 
 ![StoragePoolsDatasetUnlockexample1](/images/CORE/12.0/EncrytionExample1.png "Encrypted locked Datasets")
 
-The parent dataset is media. It has three child datasets. The `documents` child dataset has "inherited" the parent encryption settings and its password. The other two child datasets (`audio` and `video`) have their own passphrases. When the parent dataset is locked, all child datasets lock too.
+The parent dataset is media. 
+It has three child datasets. 
+The `documents` child datasetinherits the parent encryption settings and its password. 
+The other two child datasets (`audio` and `video`) have their own passphrases. 
+After locking the parent dataset all child datasets lock too.
 
 ![StoragePoolsDatasetUnlockexample2](/images/CORE/12.0/EncrytionExample2.png "Password for locked Datasets")
 
-Open the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> for the parent dataset and select *unlock*. To unlock all the datasets, check the *Unlock Children* and enter the passphrase for each dataset that needs to be unlocked.
+Open the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> for the parent dataset and click **Unlock**. 
+To unlock all the datasets, click on the **Unlock Children** checkbox to check-mark it and type the passphrase for each dataset to unlock them.
 
 ![StoragePoolsDatasetUnlockexample3](/images/CORE/12.0/EncrytionExample3.png "Successfully unlocked Datasets")
 
-Click the *Continue* button in the dialog window that confirms that the unlocking was successful. The dataset listing changes to show the unlocked icon.
+Click the **CONTINUE** button on both confirmation windows to successfully unlock the datasets. 
+The datasets display the unlocked icon.
+## Encrypting a Zvol
```

Comment:
Note particularly necessary for the final rendered article but suggested an empty line at the end of the section as a readability improvement for those looking at the raw text.
```suggestion
The datasets display the unlocked icon.

## Encrypting a Zvol
```